### ISSUE TITLE
.SendPaginatorAsync() error

### ISSUE THREAD

**Issue by Sh00ter0** [Playwo/Discord.InteractivityAddon#26]
Hello. There is a problem with **.SendPaginatorAsync()** working in the latest version of Discord.net. Does anyone know the solution to this problem?

**Discord.NET:** 3.0.0-dev-20210601.2
```
System.MissingMethodException: Method not found: 'Void Discord.WebSocket.BaseSocketClient.add_ReactionAdded(System.Func4<Discord.Cacheable2<Discord.IUserMessage,UInt64>,Discord.WebSocket.ISocketMessageChannel,Discord.WebSocket.SocketReaction,System.Threading.Tasks.Task>)'.
   at Interactivity.InteractivityService.SendPaginatorAsync(Paginator paginator, IMessageChannel channel, Nullable1 timeout, IUserMessage message, Nullable1 runOnGateway, CancellationToken cancellationToken)
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine)
   at Interactivity.InteractivityService.SendPaginatorAsync(Paginator paginator, IMessageChannel channel, Nullable1 timeout, IUserMessage message, Nullable1 runOnGateway, CancellationToken cancellationToken)
   at ULTIMATEDOGO.Modules.BotOwnerCommands.BotErrorList(String filter) in ...\Modules\BotOwnerCommands.cs:line 126
```

**Comment by Playwo** [comment]
This might be due to some changes in DNet 3.0. Will give it a try and maybe create a new prerelease later today

**Comment by Sh00ter0** [comment]
Problem fixed with version: 2.4.1-preview-20210624.1 

Thanks for your help @Playwo.
